lib/momentum/opsworks.rb

```diff
@@ -77,6 +77,27 @@ def initialize(aws_id, aws_secret)
       @ow = Momentum::OpsWorks.client(aws_id, aws_secret)
     end
 
+    def execute_recipe!(stack_name, layer, recipe, app_name = Momentum.config[:app_base_name])
```

Comment:
This is a significantly new responsibility for the `Deployer` class. If it benefits from sharing some `Deployer` behaviors (and I think it does), maybe rename it `Commander` or something?